API/API.mli

```diff
@@ -506,7 +492,6 @@ sig
       val get_type : ('c, 't) pt -> 't
 
       (** Set the identifier that is bound by a given declaration. *)
-      val set_id : Names.Id.t -> ('c, 't) pt -> ('c, 't) pt
```

Comment:
I think that this function is useful, should be kept in the API, and `API/PROPERTIES` should be updated. The update of `API/PROPERTIES` is part of [the plan](https://github.com/matejkosik/coq/blob/trunk%2BAPI_ROADMAP/API/ROADMAP).